activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -130,9 +130,9 @@ def calculate(operation, column_name)
     # the plucked column names, if they can be deduced. Plucking an SQL fragment
     # returns String values by default.
     #
-    #   Person.pluck(:id)
-    #   # SELECT people.id FROM people
-    #   # => [1, 2, 3]
+    #   Person.pluck(:name)
+    #   # SELECT people.name FROM people
+    #   # => ['David', 'Jeremy', 'Jose']
```

Comment:
:+1: new example looks good. As I myself was not aware of that method :blush: I think it's worth to include a NOTE referring the user to `.ids` if he is only looking for record ids.